main.js

```diff
@@ -1 +1,21 @@
+var submitForm = document.querySelector(".activity-button");
+var formView = document.querySelector(".left-section");
+var timerView = document.querySelector(".timer-container");
+var activityInput = document.querySelector(".text-field-long");
+var timeInput = document.querySelector(".time-inputs-sections")
+var categoryName = document.querySelector(".category-name");
 
+submitForm.addEventListener("click", submitActivity);
+
+function submitActivity(event) {
```

Comment:
We should also add code to change the "New Activity" header to "Current Activity" per the project spec.

Reply:
+1 
add on line 6.5:
`var leftSectionHeader = document.querySelector("#left-section-header")`

Reply:
added that line to 6.5. Didn't change html because I have a feeling you probably did for the timer page, Nate! :)